core/ts/__tests__/MaciState.test.ts

```diff
@@ -1,9 +1,18 @@
-import { MaciState, packProcessMessageSmallVals, unpackProcessMessageSmallVals } from "../";
+import { existsSync, readFileSync, unlinkSync, writeFileSync } from "fs";
+
 import { expect } from "chai";
-import { PCommand, Message, Keypair, VerifyingKey, StateLeaf } from "maci-domainobjs";
 
-import { hash5, G1Point, G2Point, NOTHING_UP_MY_SLEEVE, IncrementalQuinTree } from "maci-crypto";
-import { existsSync, readFileSync, unlinkSync, writeFileSync } from "fs";
+import { PCommand, Message, Keypair, StateLeaf } from "maci-domainobjs";
+import { hash5, NOTHING_UP_MY_SLEEVE, IncrementalQuinTree, AccQueue } from "maci-crypto";
+
+import {
+  STATE_TREE_ARITY,
+  STATE_TREE_SUBDEPTH,
+  BlankStateLeafHash,
+  MaciState,
+  packProcessMessageSmallVals,
+  unpackProcessMessageSmallVals,
+} from "../";
```

Comment:
Hmm, I've been importing directly from the actual path when it's within the same package, but for external packages, I thought referencing from index.ts was the standard approach. I've also treated test files as external, even though they're in the same package directory. However, I'm not entirely certain about the most idiomatic way to handle imports in JS🤔

Reply:
@0xmad what's your take here on what's the most appropriate way to deal with imports?

Reply:
Here we need to be careful because index import usually produce cycle dependencies. From my experience, we had test forlder for each `module/component` and just imported from index file.